menu/cbs/menu_cbs_ok.c

```diff
@@ -1152,7 +1152,19 @@ int generic_action_ok_displaylist_push(const char *path,
             dl_type            = DISPLAYLIST_FILE_BROWSER_SELECT_FILE;
 
             /* Focus on current content entry */
-            menu_driver_set_pending_selection(path_basename(path_get(RARCH_PATH_CONTENT)));
+            {
+               char path_content[PATH_MAX_LENGTH];
+               path_content[0] = '\0';
```

Comment:
I see this quite often preceeding strlcpys and snprintfs within the libretro codebase. This is redundant in those cases and any half-smart compiler will optimize that without translating into machine code.